snappy_pipeline/workflows/hla_typing/model.py

```diff
@@ -28,6 +28,6 @@ class HlaTyping(SnappyStepModel, validators.ToolsMixin, validators.NgsMappingMix
 
     tools: Annotated[list[Tool], EnumField(Tool, [Tool.optitype], min_length=1)]
 
-    optitype: Optitype | None = None
+    optitype: Optitype = Optitype()
```

Comment:
I am tempted to always provide defaults. It makes the `__init__.py` code easier, and I personally don't like to see:

```
step_config:
   hla_typing:
    tools: [optitype]
    optitype: {}
```

For me, the syntax `optitype: {}` suggests that the tool doesn't accept parameters, _not_ that we use the defaults.
Also, I wouldn't always set all defaults for more complex steps/tools. If there isn't an obvious choice for a default, the the user should make a decision.

But again, I am prepared to be convinced otherwise.